Cargo.toml

```diff
@@ -1,16 +1,21 @@
 [package]
 name = "libtransport-tcp"
 version = "0.0.1"
-authors = ["Samuel Marks <@SamuelMarks>"]
+authors = [
+    "Maxim Zakharov <@Maxime2>",
+    "Samuel Marks <@SamuelMarks>",
+    "Vladimir Komendantskiy <komendantsky@gmail.com>",
+]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 bincode = "1.1.4"
+buffer = "0.1.8"
+futures-preview = { version = "=0.3.0-alpha.17" }
 libcommon-rs = { git = "https://github.com/Fantom-foundation/libcommon-rs" }
 libtransport = { git = "https://github.com/Fantom-foundation/libtransport" }
 serde = "1.0.98"
 serde_derive = "1.0.98"
-buffer = "0.1.8"
-futures-preview = { version = "=0.3.0-alpha.17", features = ["compat"] }
+tokio = "0.2.0-alpha.1"
```

Comment:
Changing from standard Rust library to 3rd-party crates is out of the scope of this repo. If there is a believe it would work faster implemented using tokio crate then a separate implementation of libtransport should be created.

Reply:
I'll remove it.